specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-10-27-preview/privateAccesses.json

```diff
@@ -0,0 +1,580 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update a chaos private access instance": {
+            "$ref": "./examples/PrivateAccessPatchUpdate.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "PrivateAccesses"
+        ],
+        "operationId": "PrivateAccesses_CreateOrUpdate",
+        "description": "Create or update a private access",
+        "parameters": [
+          {
+            "$ref": "./types/common.json#/parameters/apiVersion"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/PrivateAccessName"
+          },
+          {
+            "name": "privateAccess",
+            "description": "private access resource to be created or updated.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./types/privateAccesses.json#/definitions/PrivateAccess"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Private access resource created successfully.",
+            "schema": {
+              "$ref": "./types/privateAccesses.json#/definitions/PrivateAccess"
+            }
+          },
+          "200": {
+            "description": "Private Access resource updated successfully.",
+            "schema": {
+              "$ref": "./types/privateAccesses.json#/definitions/PrivateAccess"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create or Update a private access resource": {
+            "$ref": "./examples/CreateOrUpdateAPrivateAccess.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "PrivateAccesses"
+        ],
+        "operationId": "PrivateAccesses_Delete",
+        "description": "Delete a private access",
+        "parameters": [
+          {
+            "$ref": "./types/common.json#/parameters/apiVersion"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/PrivateAccessName"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
what scenario do you return 200 vs 204 here?

Reply:
This should only be 204, no 200.